core/locals.tf

```diff
@@ -14,6 +14,9 @@ locals {
   # The application exposed port
   app_port = 3000
 
+  # The RDS port
+  rds_port = 5432
```

Comment:
is it required? we have port setting per env

Reply:
Nice catch!!! It was updated to use the port from rds config in 7f097a5 ✅